### ISSUE TITLE
Navigating to a section within a note via a link

### ISSUE THREAD

**Issue by DenisVDavydov** [dy-sh/obsidian-remember-cursor-position#64]
The plugin performs its function well. However, it would be preferable if the plugin did not interfere with navigating to a specific section within a note. For example, transitioning from note #A to note #B (to section ### B.10 via a link) should work seamlessly, even if the previous time note #B was opened at section B.2.

Here’s how it currently works: 1) note B is opened; 2) it's viewed up to section B.2; 3) then closed; 4) note A is opened; 5) clicking a link in note A should lead to section B.10 in note B; 6) instead, it returns to section B.2, not B.10 as expected.

**Comment by MyMarvel** [comment]
I agree. I found a weird settings about this (a percentage slider without visible values), so I assume the developer knows about this problem, but the slider doesn't work - I moved it to the maximum and still the same behaviour described above appears. Obsidian Desktop Linux 1.7.6 We need help. I had to disable this plugin unfortunately.
![image](https://github.com/user-attachments/assets/4f6b6f8e-4934-4b56-bd96-42cc371f3354)
